azure-pipelines.yml

```diff
@@ -127,7 +127,7 @@ jobs:
           display_name: "Linux"
           go_path: '$(go env GOPATH)/bin/'
           suffix: ''
-          run_e2e: 'go test -timeout=1h -v ./e2etest | tee test.txt'
+          run_e2e: 'go test -timeout=2h -v ./e2etest | tee test.txt'
```

Comment:
oh no /s

Reply:
yeah our e2e tests are getting out of control